interfaces/Portal/src/app/program/make-payment/make-payment.component.ts

```diff
@@ -194,8 +194,12 @@ export class MakePaymentComponent implements OnInit, OnDestroy {
   }
 
   private onPaymentError(error) {
-    if (error && error.error && error.error.errors) {
-      actionResult(this.alertController, this.translate, error.error.errors);
+    if (error && error.error && (error.error.errors || error.error.message)) {
```

Comment:
I added this to ensure the error message is visible in the portal. The code is not pretty as I don't really know when `error.error.errors` occurs. Do you think it's worth the time to dive into this further?  @jannisvisser